src/status.rs

```diff
@@ -79,11 +79,11 @@ pub struct RadioShow {
 }
 
 /// The main Space API status object.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, RustcDecodable)]
 pub struct Status {
 
     // Hackerspace properties
-    pub api: &'static str,
```

Comment:
I would like to change it, because the rustc_serialize does not work with `&'static str`

    the trait `rustc_serialize::serialize::Decodable` is not implemented for the type `&str` [E0277]
    src/status.rs:82 #[derive(Debug, Clone, RustcDecodable)]